homeassistant/components/mailbox/__init__.py

```diff
@@ -84,7 +87,7 @@ async def async_setup_platform(p_type, p_config=None, discovery_info=None):
         await component.async_add_entities([mailbox_entity])
 
     setup_tasks = [
-        asyncio.create_task(async_setup_platform(p_type, p_config))
+        asyncio.create_task(async_setup_platform(p_type, p_config))  # type: ignore[no-untyped-call]
```

Comment:
I couldn't work out the type hints should be in this local version of `async_setup_platform`.
I'm happy to add if someone gives me pointers here, but I think it's already nice to have the few added hints already.

Reply:
The issue is here: <https://github.com/home-assistant/core/blob/7d1ae5651467988f6550ec93d164d2742d10cc66/homeassistant/helpers/__init__.py#L14:5>

Which should return `-> Iterable[tuple[str, ConfigType]]: